components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleOAuth2Authenticator.java

```diff
@@ -58,6 +70,158 @@ private void initTokenEndpoint() {
         }
     }
 
+    @Override
+    public boolean canHandle(HttpServletRequest request) {
+
+        boolean canHandle = false;
+        if (log.isDebugEnabled()) {
+            log.debug("Inside OpenIDConnectAuthenticator.canHandle()");
+        }
+        if (StringUtils.isNotBlank(request.getParameter("oneTapEnabled")) &&
+                request.getParameter("oneTapEnabled").equals("YES")) {
```

Comment:
Here if we change the sides of the equals operation, it will resolve the null pointer and the equals operation both right?
```
"Yes".equals(request.getParameter("oneTapEnabled"))
```
WDYT?

Reply:
Fixed f2aa0c1666f9b5234e7d113125fadd55008081cd